pkg/storage/test/tuples.go

```diff
@@ -1498,6 +1498,54 @@ func ReadStartingWithUserTest(t *testing.T, datastore storage.OpenFGADatastore)
 		}
 		require.Equal(t, []string{"doc1", "doc3", "doc4", "doc5", "doc6", "doc7"}, actualObjectIDs)
 	})
+	t.Run("enforce_order_of_tuples_with_public_wildcard", func(t *testing.T) {
```

Comment:
it's not clear to me, looking at line 1547, what exactly the result is ordered on.

Reply:
Added